bodhi/server/models.py

```diff
@@ -476,12 +476,28 @@ class TestCase(Base):
 
 
 class Package(Base):
+    """
+    This model represents a package.
+
+    This model uses single-table inheritance to allow for different package types.
+
+    Attributes:
+        name (unicode): A unicode string that uniquely identifies the package.
+        requirements (unicode): A unicode string that describes the requirements of a package.
```

Comment:
Perhaps "A unicode string that lists space-separated taskotron test results that must pass for this package."